classes/fields/pick.php

```diff
@@ -2389,6 +2391,28 @@ public function is_autocomplete( $options ) {
 		return $autocomplete;
 	}
 
+	/**
+	 * Check if a field supports AJAX mode
+	 *
+	 * @param string $type
+	 * @param array  $options Field options.
+	 *
+	 * @return bool
+	 * @since 2.7.4
+	 */
+	public function can_ajax( $type, $options ) {
```

Comment:
This method can be simplified further down to:

Reply:
But that still leaves half the method being about determining if this is simple tableless (whatever that might mean), so let's extract that logic into its own method as well:

```php
public function is_simple_tableless( $type, $options ) {
	$field_object = pods_v( $type . '_object', $options );
	
	return in_array( $field_object, PodsForm::simple_tableless_objects(), true );	
}

public function can_ajax( $type, $options ) {
	return $this->is_autocomplete( $options ) && ! $this->is_simple_tableless( $type, $options );
}
```

The first method could be made `protected` if it's not needed to be public.

Reply:
This will need a fresh issue since I already merged this PR.  

I actually want to set both those methods as well `is_autocomplete()` (only added in 2.7) to private, crossed my mind but I forgot about it.  I'm not sure those methods survive a proper full refactor and anything public becomes a back-compat liability until the end of time.

Reply:
> This will need a fresh issue since I already merged this PR.

Done in #4993 + #4994.